### PR TITLE
Delete Forums article

### PR DESCRIPTION
**Pull request by MegaApplePi**

### CODE REVIEW

**Comment by MegaApplePi** [comment]
- [ ] Rewrite subforum descriptions

File: wiki/Forums/en.md [hunk 1]
```diff
@@ -1,76 +1,76 @@
...
-    - [Türkçe](https://osu.ppy.sh/community/forums/93): _Turkish_
-    - [ภาษาไทย](https://osu.ppy.sh/community/forums/54): _Thai_
-    - [日本語](https://osu.ppy.sh/community/forums/32): _Japanese_
+- [osu!](/community/forums/1)
+  - [Development](/community/forums/2): Discussion for projects, considerations, news and publicity, meetings and the likes.
+    - [Completed Projects](/community/forums/116): Discussion for completed or archived projects.
+      - [osu! iPhone](/community/forums/47)
+        - [Bug Reports](/community/forums/50)
+    - [Community Voice](/community/forums/65): Polls to help the team understand the osu! community a bit better.
+    - [Ranking Criteria](/community/forums/87): Discussion for what will and will not be allow in ranked beatmaps.
```
Comment (Joehuu) [inline wiki/Forums/en.md]: `allow` -> `allowed`

File: wiki/Forums/en.md [hunk 2]
```diff
@@ -1,76 +1,76 @@
...
+  - [Gameplay & Rankings](/community/forums/13): Discuss your scores, strategies etc. Anything about playing osu!standard goes here.
+    - [Tournaments](/community/forums/55): A place to announce and record your active or completed tournaments.
+    - [Mapping Techniques](/community/forums/61): Discuss breathtaking or unimpressive hit object positioning techniques.
+    - [Taiko](/community/forums/105): Anything about playing osu!taiko goes here.
+    - [Catch the Beat](/community/forums/106): Anything about playing osu!catch goes here.
+    - [osu!mania](/community/forums/107): Anything about playing osu!mania goes here.
+  - [Skinning](/community/forums/15): Release your completed or WIP skins! You can also have general discussion or ask questions about skinning here.
+    - [Work In Progress Skins](/community/forums/119)
+    - [Completed Skins](/community/forums/109): Listing and discussion for completed skin threads.
+  - [Feature Requests](/community/forums/4): Give feedback and suggestions for what you would like to see in osu!
```
Comment (Joehuu) [inline wiki/Forums/en.md]: Missing period.

File: wiki/Forums/en.md [hunk 3]
```diff
@@ -1,76 +1,76 @@
...
+  - [Beatmap Management](/community/forums/115): A place for Beatmap Nominators to discuss.
+    - [Archive](/community/forums/117)
+- [Other](/community/forums/11)
+  - [General Discussion](/community/forums/7): Post about anything non-osu!, as long as it's quality content.
+  - [Off-Topic](/community/forums/52): For the not-so-logical discussions.
+    - [Forum Games](/community/forums/68): A forum of forum games!
+      - [Mafia](/community/forums/84): Mafia, Werewolf, and other similar games can be found here!
+    - [Surveys](/community/forums/114): PRETEND THIS IS ASK.FM AND ASK PEOPLE QUESTIONS THAT NOBODY CARES ABOUT
+  - [Introductions](/community/forums/8): Introduce yourself!
+  - [Music Hall](/community/forums/91): Music discussion outside the realm of osu!
```
Comment (Joehuu) [inline wiki/Forums/en.md]: Missing period.

File: wiki/Forums/en.md [hunk 4]
```diff
@@ -1,76 +1,76 @@
...
+  - [Skinning](/community/forums/15): Release your completed or WIP skins! You can also have general discussion or ask questions about skinning here.
+    - [Work In Progress Skins](/community/forums/119)
+    - [Completed Skins](/community/forums/109): Listing and discussion for completed skin threads.
+  - [Feature Requests](/community/forums/4): Give feedback and suggestions for what you would like to see in osu!
+    - [Completed Requests](/community/forums/30): Fulfilled or discontented requests lie in peace here.
+  - [Help](/community/forums/5): The place to go when you have a problem to report or a question to ask that doesn't go anywhere else.
+    - [Resolved Issues](/community/forums/29): All resolved issues go here.
+    - [Confirmed Issues](/community/forums/101): List of confirmed issues.
+- [Beatmaps](/community/forums/2)
+  - [Ranked/Approved Beatmaps](/community/forums/14): Officially approved beatmaps reside here.
```
Comment (Joehuu) [inline wiki/Forums/en.md]: `approved` -> `ranked`

File: wiki/Forums/en.md [hunk 5]
```diff
@@ -1,76 +1,76 @@
...
-    - [日本語](https://osu.ppy.sh/community/forums/32): _Japanese_
+- [osu!](/community/forums/1)
+  - [Development](/community/forums/2): Discussion for projects, considerations, news and publicity, meetings and the likes.
+    - [Completed Projects](/community/forums/116): Discussion for completed or archived projects.
+      - [osu! iPhone](/community/forums/47)
+        - [Bug Reports](/community/forums/50)
+    - [Community Voice](/community/forums/65): Polls to help the team understand the osu! community a bit better.
+    - [Ranking Criteria](/community/forums/87): Discussion for what will and will not be allow in ranked beatmaps.
+      - [Finalized/Denied Amendments](/community/forums/92): A place for dead Ranking Criteria proposals and amendments.
+    - [osu!droid](/community/forums/85)
```
Comment (Joehuu) [inline wiki/Forums/en.md]: The `osu!droid` forum exists, but doesn't show it on forums anymore.

**Comment by Joehuu** [comment]
The image should be updated to the new header. Wait for ppy/osu-web#3377 to get merged.

It should also have a timestamp since it is the type of page that updates. Or change the forum names and usernames to examples.

**Comment by MegaApplePi** [comment]
@Joehuu 

> The image should be updated to the new header. Wait for ppy/osu-web#3377 to get merged.

This pull request *was* intended make changes to everything else but that image, but alright, I'll wait to update it.

> It should also have a timestamp since it is the type of page that updates.

It's just a screenshot, does it really matter that much? Timestamping images is quite excessive since you can just look at the commit history of the file to see when it was last changed and see how it looked previously.

> Or change the forum names and usernames to examples.

That seems a bit overkill but alright, will add lorem ipsum.

**Comment by peppy** [comment]
These kinds of pages should *really* be avoided. You're creating a second list of another list already on the site for no apparent purpose.

I could understand if you include hte main forum sections along with longer information on what they contain, but listing every single forum is just wrong and will mean this articles requires constant attention to keep it -up-to-date (for what, though?)

**Comment by MegaApplePi** [comment]
Alright then, will delete.

**Pull request merged**